hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -360,10 +356,9 @@ protected boolean needsBalance(Cluster cluster) {
   }
 
   @Override
-  public synchronized List<RegionPlan> balanceCluster(TableName tableName, Map<ServerName,
-    List<RegionInfo>> clusterState) {
-    this.tableName = tableName;
-    return balanceCluster(clusterState);
+  public synchronized List<RegionPlan> balanceCluster(Map<TableName, Map<ServerName,
+    List<RegionInfo>>> clusterLoadPerTable) {
+    return super.balanceCluster(clusterLoadPerTable);
```

Comment:
If call super method directly, no need to add this method here?

Reply:
yes, I remove it